entity-framework/toc.yml

```
@@ -20,6 +20,8 @@
     items:
     - name: Overview
       href: core/index.md
+    - name: Take the EF Core 6 survey
```

```suggestion
    - name: Take the EF Core 6 survey >>
```
For consistency perhaps. We append it when clicking on it would navigate away from ef docs.